src/createReconciler.spec.lua

```diff
@@ -302,7 +302,9 @@ return function()
 			local node = noopReconciler.mountVirtualNode(emptyFragment, nil, "test")
 
 			expect(node).to.be.ok()
-			expect(next(node.children)).to.never.be.ok()
+
+			local nextNode = next(node.children)
```

Comment:
why was this extraction necessary? to force/avoid widening? if so, please mark with `-- FIXME Luau`.

Reply:
Probably because `next` returns multiple values and `expect` only accepts 1

Reply:
So, a Luau issue then? I'll ask in the slack